flow/src/main/java/flow/Flow.java

```diff
@@ -126,6 +124,10 @@ private Traversal(@Nullable History from, History to, Direction direction,
     public Context createContext(Object key, Context baseContext) {
       return new FlowContextWrapper(keyManager.findServices(key), baseContext);
     }
+
+    public State getState(Object key) {
```

Comment:
Works for me. Curious about the usecase for something other than getting the top state?